loc/translations-import/vscode-powerplatform.fr.xlf

```diff
@@ -145,6 +145,46 @@ L’authentification interactive n’est pas disponible pour les scénarios dist
         <source xml:lang="en">PAC CLI is not supported in VS Code for the Web</source>
         <target state="translated">PAC CLI n’est pas pris en charge dans VS Code pour le Web</target>
       </trans-unit>
+      <trans-unit id="microsoft-powerapps-portals.walkthrough.title">
```

Comment:
Looks like there are a decent number of strings in the `package.json` section of the translated xlf files here that don't have translations added.

I _think_ that should be fine, as these are used by the Web Only version of the extension, which currently doesn't load translations anyway due to a VS Code Web issue.   @ashishchoudhary001 , can you confirm that?

Reply:
Resolving comment per Hiro's reply below - these strings will be localized later this week.